.gitattributes

```diff
@@ -0,0 +1,2 @@
+infra/** linguist-vendored
```

Comment:
Can you also add these https://github.com/bemanproject/optional/blob/main/.gitattributes?

We should also exclude LaTeX

Reply:
Done.

> We should also exclude LaTeX

How should we go about that? Should I exclude all `*.tex` files?

Reply:
Yes, now it's good